lang/java/reef-common/src/main/java/org/apache/reef/client/DriverLauncher.java

```diff
@@ -59,15 +60,21 @@
       .build();
 
   private final REEF reef;
+  private final UserCredentials user;
 
   private LauncherStatus status = LauncherStatus.INIT;
 
   private String jobId;
   private RunningJob theJob;
 
   @Inject
-  private DriverLauncher(final REEF reef) {
+  private DriverLauncher(final REEF reef, final UserCredentials user) {
```

Comment:
When the DriverLauner get injected, the UserCredentials  also got injected, right? Does it contain any tokens or just an newly injected YarnProxyUser?